Dockerfile.build

```diff
@@ -1,6 +1,7 @@
 FROM ubuntu:20.04
 
-RUN apt-get update && apt-get install -y \
+RUN apt-get update && apt-get install -y  --no-install-recommends \
+    build-essential \
```

Comment:
The `--no-install-recommends` option only fixes this for a fairly indirect reason; it will break again if any of the truly necessary packages start asking questions.

Per https://manpages.debian.org/jessie/debconf-doc/debconf.7.en.html it seems that the way to opt out of interactivity once for all is to use `DEBIAN_FRONTEND=noninteractive apt-get install` .